app/controllers/issue_view_columns_controller.rb

```diff
@@ -25,6 +25,6 @@ def update
       c.save!
     end
 
-    redirect_to settings_project_path(@project, tab: 'issue_view_columns'), notice: l(:label_issue_columns_created_sucessfully)
+    redirect_to settings_project_path(@project, tab: "issue_view_columns"), notice: l(:label_issue_columns_created_sucessfully)
```

Comment:
Why transforming the `'` to `"`? Is this an automated process? If so, can you disable it and revert these types of modifications?